documentation/howtos.md

```diff
@@ -4,61 +4,153 @@ By default, instances are not provisioned with either ecommerce or the Course Di
+     --partner-code 'partn_id' \ # this is limited to only a few characters
+     --partner-name 'Partner Name' \
+     --lms-url-root 'https://lms.external.domain' \
+     --client-id '{ecommerce_worker oauth client id}' \
+     --client-secret '{ecommerce_worker oauth client secret}' \
+     --from-email 'noreply@todo.lms.external.domain' \ # TODO: what should this be?
+     --discovery_api_url 'https://discovery.external.lms.domain/api/v1'
+   ```
+
+   If using the Stripe payment processor, then it must be configured as a
+   client side payment processor - either add `--client-side-payment-processor
+   stripe` to the above command or enter it later from Django admin. See also
+   [docs](https://edx-ecommerce.readthedocs.io/en/latest/additional_features/payment_processors.html)
+   for specific payment processors.
+
+   Also note that any of the values from the `create_or_update_site` command
+   can be edited later in Django admin at `/admin/core/siteconfiguration/`; no
+   need to continue rerunning this command once the initial auth related setup
+   is working for the site configuration.
 1. In the discovery env, configure a partner using
    [`create_or_update_partner`](https://github.com/edx/course-discovery/blob/master/course_discovery/apps/core/management/commands/create_or_update_partner.py).
    Use the same partner code as what you used for ecommerce.
+
+   ```
+   sudo -u discovery -Hs
+   cd
+   source discovery_env
+   source venvs/discovery/bin/activate
+   cd discovery
+   python manage.py create_or_update_partner \
+     --site-id 1 \
+     --site-domain 'discovery.external.lms.domain' \
+     --code 'partn_id' \
+     --name 'Client Name' \
+     --courses-api-url 'https://external.lms.domain/api/courses/v1/' \
+     --ecommerce-api-url 'https://ecommerce.external.lms.domain/api/v2/' \
+     --organizations-api-url 'https://external.lms.domain/api/organizations/v0/' \
+     --oidc-url-root 'https://external.lms.domain/oauth2' \
+     --oidc-key '{discovery oauth client id}' \
+     --oidc-secret '{discovery oauth client secret}'
+   ```
 1. [Configure LMS to use ecommerce](http://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/ecommerce/install_ecommerce.html#switch-from-shoppingcart-to-e-commerce)
 
 Test your configuration:
 
 1. Verify the OIDC login for both services:
 
-        https://ecommerce-<your-instance>.opencraft.hosting/login
-        https://discovery-<your-instance>.opencraft.hosting/login
+        https://ecommerce.<your-instance>.opencraft.hosting/login
+        https://discovery.<your-instance>.opencraft.hosting/login
 
 1. Verify that the discovery cronjob runs without errors:
 
-        sudo -s -c . /edx/app/discovery/discovery_env; /edx/bin/manage.discovery refresh_course_metadata
+        sudo -u discovery -Hs
+        source /edx/app/discovery/discovery_env
+        /edx/bin/manage.discovery refresh_course_metadata
+
+TODO: add more complete manual test checklist
```

Comment:
1. Create a new course to test ecommerce.
1. Add a new [Course seat](https://edx-ecommerce.readthedocs.io/en/latest/create_products/create_course_seats.html) for this course, with a nominal charge (e.g. $1).
1. Optionally add a [Coupon](https://edx-ecommerce.readthedocs.io/en/latest/create_products/create_coupons.html) for the course, if the client will use coupons.
1. Enroll a new user into the test course, and verify that the coupon works, payments are processed, and the enrollment succeeds.
   Ideally, the payment processor will be initially configured as a sandbox service for testing, and so you can use test credit card numbers as provided by the payment processor.

Reply:
Awesome, thanks! :D